knees4bees/rancid-tomatillos
# Feature/movie trailer

## Is this a fix, feature, or refactor?
This is a feature

## What is the change?
* The user can now play trailers on the movie details page
* Cleaning functions have been added. 

## What does it fix?
Failing test were refactored and now pass

## Where should the reviewer start?
Review the new test added to cypress. Review the new Trailer component.

## How should this be tested?
Run cypress locally to make sure all test are passing. 
Travel to a movie's details and click the watch trailer button to view trailer.


## Relevant ScreenShots
